akka-docs/rst/java/code/docs/testkit/ParentChildTest.java

```diff
@@ -45,7 +45,7 @@ public Receive createReceive() {
     public Receive createReceive() {
       return receiveBuilder()
         .matchEquals("ping", message -> {
-          getContext().parent().tell("pong", self());
+          getContext().getParent().tell("pong", self());
```

Comment:
I disagree. I had the same discussion with Konrad the other day when he reviewed the ref-card work I did. In that case we should never use the `get` style accessors in Java samples because what makes it a `getContext()` but a `self()` or `parent()`? It's not a question of a fluent API here, it _is_ an accessor for a field. 

I think the Scala-style accessors look neat in our Scala-eyes, but that for a Java dev it is very weird and unexpected. As far as I know there has never been a move away from getters/setters for fields in the Java world?

I'd rather see that we consequently used the getters for Java samples, and let the (I'd argue few) devs who likes the Scala-style do that.